content/blog/newsletter_2022-02-05.md

```diff
@@ -0,0 +1,236 @@
+
+Webhook payloads are processed by middleware and the XMPP notifications are template-based and written in [EJS](https://ejs.co/ "Embedded JavaScript templating"). It currently comes with GitLab and plain Git (yes, plain Git) integrations as well as a crude and untested Slack middleware, but it also understands plain text and PNG, JPEG and PDF content, which is sent to subscribers as attachments via [HTTP File Upload (XEP-0363)](https://xmpp.org/extensions/xep-0363.html).
+
+The code is far from production quality but the developer would welcome any and all comments and suggestions to inform further development. The main repository is hosted at https://gitlab.com/navlost.eu/xmpp/components/webhooks and there is also a [demo server](https://gitlab.com/navlost.eu/xmpp/components/webhooks/-/blob/devel/README.md#demo-server) available for casual testing.
+
+[Libervia 0.8 « La Cecília »](https://www.goffi.org/b/libervia-v0-8-la-cecilia-BdQ4) (formerly known as « Salut à Toi ») has been released with a complete OMEMO encrytion finalization for group chats, a new default theme, an easy to use invitation system, a non-standard (XMPP) list feature, photo albums and many technical changes.
+
+A new stable release of [SiskinIM 7.0.1 has been published](https://github.com/tigase/siskin-im/releases/tag/7.0.1) which includes sending unencrypted messages in single chats with default encryption for OMEMO and presenting automatic file download size limit.
+
+## Servers
+
+[Openfire 4.7.0](https://discourse.igniterealtime.org/t/openfire-4-7-0-has-been-released/91268) has been released. This release is the first non-patch release in more than a year, which brings a healthy amount of new features, as well as bug fixes. Highlights of this release include extensively improved clustering support, particularly around Multi-User Chat functionality, which should benefit high-volume environments. Previously also [Openfire 4.5.5](https://discourse.igniterealtime.org/t/openfire-4-6-6-and-4-5-5-releases-log4j-only-changes/91139), [Openfire 4.6.5](https://discourse.igniterealtime.org/t/openfire-4-6-5-released/91108) and [Openfire 4.6.6](https://discourse.igniterealtime.org/t/openfire-4-6-6-and-4-5-5-releases-log4j-only-changes/91139) have been released.
+
+[Prosody 0.11.13](https://blog.prosody.im/prosody-0.11.13-released/) has been released. Since december, the new prosody releases brought some fixes to PEP to control memory usage, a security fix that addresses a denial-of-service vulnerability in Prosody’s mod_websocket, and a fix for a memory leak. Previously [Prosody 0.11.11](https://blog.prosody.im/prosody-0.11.11-released/] and [Prosody 0.11.12](https://blog.prosody.im/prosody-0.11.12-released/) were released, too.
```

Comment:
```suggestion
[Prosody 0.11.13](https://blog.prosody.im/prosody-0.11.13-released/) has been released. Since December, new Prosody releases brought some fixes to PEP to control memory usage, a security fix that addresses a denial-of-service vulnerability in Prosody’s mod_websocket, and a fix for a memory leak. Previously [Prosody 0.11.11](https://blog.prosody.im/prosody-0.11.11-released/] and [Prosody 0.11.12](https://blog.prosody.im/prosody-0.11.12-released/) were released, too.
```